lib/Swagger/src/User/Preferences/Api/UserPreferencesApi.php

```diff
@@ -0,0 +1,342 @@
+<?php
+/**
+ *  Copyright 2015 SmartBear Software
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+/**
+ *
+ * NOTE: This class is auto generated by the swagger code generator program. Do not edit the class manually.
+ */
+
+namespace Swagger\Client\User\Preferences\Api;
+
+use \Swagger\Client\Configuration;
+use \Swagger\Client\ApiClient;
+use \Swagger\Client\ApiException;
+use \Swagger\Client\ObjectSerializer;
+
+class UserPreferencesApi {
+
+  /** @var \Swagger\Client\ApiClient instance of the ApiClient */
+  private $apiClient;
+
+  /**
+   * @param \Swagger\Client\ApiClient|null $apiClient The api client to use
+   */
+  function __construct($apiClient = null) {
+    if ($apiClient == null) {
+      $apiClient = new ApiClient();
+      $apiClient->getConfig()->setHost('https://localhost/user-preference');
```

Comment:
No. This is autogenerated code, which gets the `basePath` from the swagger documentation, which doesn't specify a domain, so it defaults to `localhost`